pkg/authn/signingMethod.go

```diff
@@ -7,14 +7,14 @@ import (
 
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 	"github.com/golang-jwt/jwt/v5"
+	"github.com/xmtp/xmtpd/pkg/utils"
 )
 
 const (
-	ALGORITHM               = "ES256K"
-	SIG_LENGTH              = 65
-	R_LENGTH                = 32
-	S_LENGTH                = 32
-	DOMAIN_SEPARATION_LABEL = "jwt"
+	ALGORITHM  = "ES256K"
```

Comment:
Kind of fiddly at this point, but I believe the definition of `ES256K` specifies that sha256 is used as the hashing algorithm. I guess there's no JWT `ALGORITHM` that uses keccak256?

Reply:
Ya. I noticed that too.

We could make one up, but I think this is less confusing.